### PR TITLE
Add support for boolean data in coerceVariantType

### LINKED ISSUE #794: coerceVariantType on string "true"
Hi,

I am pulling values from a CSV file and running coerceVariantType() on the value before writing to the server.

Works for all except, booleans.

`
   let valueToWrite = coerceVariantType(DataType.Boolean, "true")
`

This will return a string, not true. Any support for that in the future ?

### PR DESCRIPTION
**Pull request by maneetgoyal**
Fixes #794

### CODE REVIEW

File: packages/node-opcua-variant/test/test_variant.js [hunk 1]
```diff
@@ -1460,6 +1460,10 @@ describe("miscellaneous Variant tests", () => {
         coerceVariantType(DataType.Int32, { value: 26, key: "foo" }).should.eql(26);
     });
 
+    it("coerceVariantType on Boolean", () => {
+        coerceVariantType(DataType.Boolean, true).should.eql(true);
```
Comment (erossignon) [inline packages/node-opcua-variant/test/test_variant.js:1464]: Please add a test to check that "true"|"false" can be converted to true|false.

 coerceVariantType(DataType.Boolean, "true" /* a string here  */).should.eql(true);
 
coerceVariantType(DataType.Boolean, "false" /* a string here  */).should.eql(false);

**Review by erossignon — CHANGES_REQUESTED** [review]
Please add a unit test to verify that the suggested change produces the v expected result.

File: packages/node-opcua-variant/test/test_variant.js (hunk 1 above)
Comment (maneetgoyal) [reply packages/node-opcua-variant/test/test_variant.js:1464]: On it 👍

Comment (maneetgoyal) [reply packages/node-opcua-variant/test/test_variant.js:1464]: Done, please check.

**Pull request closed without merging**